common/src/main/java/com/redhat/cloud/notifications/models/dto/v1/endpoint/EndpointDTO.java

```diff
@@ -89,6 +91,9 @@ private boolean isSubTypeNotPresentWhenNotRequired() {
         return this.type.requiresSubType || this.subType == null;
     }
 
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private Set<BundleDTO> eventTypes;
```

Comment:
![image](https://github.com/user-attachments/assets/226e618e-313b-4eb1-a4f9-04864772151d)

Reply:
UI team requested to have those event types grouped by bundle and app. But the name could be updated to something like `eventTypesByBundle` ?